src/Compilers/VisualBasic/Portable/Lowering/AsyncRewriter/AsyncRewriter.vb

```diff
@@ -292,15 +292,23 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                                          currentMethod:=currentMethod)
         End Function
 
+        ''' <returns>
+        ''' Returns true if any members that we need are missing or have use-site errors.
+        ''' </returns>
         Friend Overrides Function EnsureAllSymbolsAndSignature() As Boolean
-            Dim hasErrors As Boolean = MyBase.EnsureAllSymbolsAndSignature
+            If MyBase.EnsureAllSymbolsAndSignature Then
+                Return True
+            End If
```

Comment:
I didn't feel strong either way. From previous CR feedback from Aleksey on iterator rewriter, allows for a bit cleaner code without `hasError` carrying throughout.